src/generator/generate.ts

```diff
@@ -103,28 +107,17 @@ const getOutputSourceDir = (params: { esm: boolean }): string => {
         `Nexus Prisma error: Could not find the project root. Project root is the nearest ancestor directory to where this module is running (${__filename}) containing a package.json. Without this information Nexus Prisma does not know where to output its generated code.`
       )
     }
-    outputSourceDir = Path.join(
-      Path.dirname(packageJsonFilePath),
-      params.esm ? 'node_modules/nexus-prisma/dist-esm/runtime' : 'node_modules/nexus-prisma/dist-cjs/runtime'
-    )
+    outputSourceDir = Path.join(Path.dirname(packageJsonFilePath), 'node_modules/.nexus-prisma')
   } else {
-    /**
-     * At this point in the code we don't know if the CLI running is the CJS or ESM version.
-     * If it is the CJS version and we're doing an ESM build then we need to adjust the __dirname value.
-     */
-    outputSourceDir = Path.join(
-      params.esm ? __dirname.replace('dist-cjs', 'dist-esm') : __dirname,
-      '../runtime'
-    )
+    outputSourceDir = Path.join(__dirname, '../../../../node_modules/.nexus-prisma')
```

Comment:
Found that after like 5 hours of testing and debugging.
We must go 4 steps back because if we use `Yalc` in the testing we need to one level more to be in the project dir then we need to go inside `node_modules`

Reply:
General empathy: Yep this kind of stuff is _super_ painful.